src/driver/cockroachdb/CockroachQueryRunner.ts

```
@@ -225,7 +225,7 @@ export class CockroachQueryRunner
         } else {
             this.storeQueries = false
             this.transactionDepth -= 1
-            await this.query("RELEASE SAVEPOINT cockroach_restart")
+            // await this.query("RELEASE SAVEPOINT cockroach_restart")
```

If we have some commented out code, I think we should include some explanation of why it is commented out, or just have it deleted altogether.

---

Added a comment about it and a link to this issue

---

I think this is just hiding the real error:

https://www.cockroachlabs.com/docs/stable/release-savepoint#handling-errors

`commitTransaction` should probably not be called if there's an error. Unless the error happens exactly in it?